docs/Logging.md

```diff
@@ -49,9 +49,24 @@ and similarly for the log methods on the plugin object:
     plugin.logdebug("he just really likes turtles", connection);
 
 failing to provide a connection and/or plugin object will leave
-the default values in the log (currently "core" and
-"no\_connection").
+the default values in the log (currently "core").
 
-this is implemented by testing for argument type in
+This is implemented by testing for argument type in
 the logger.js log\* method. objects-as-arguments are then sniffed
 to try to determine if they're a connection or plugin instance.
+<<<<<<< HEAD:docs/Logging_API.md
+
+The logfmt format is also supported and can be enabled by changing the format
+from `default` to `logfmt` in the `config/log.ini` file which will
+start logging in the following format below.
+
+    level=PROTOCOL connection_uuid=9FF7F70E-5D57-435A-AAD9-EA069B6159D9.1 source=core message="S: 354 go ahead, make my day"
```

Comment:
I thought `connection_uuid` was changed to `uuid` ?

Reply:
It's still `connection_uuid`, do you want me to rename to `uuid`?

Reply:
nvm